src/ring_theory/ideal/local_ring.lean

```diff
@@ -359,6 +359,18 @@ begin
   exact map_nonunit f a ha
 end
 
+/-- Applying `map` to the identity ring homomorphism gives the identity ring homomorphism. -/
+def map_id :
```

Comment:
```suggestion
lemma map_id :
```
As the linter will point out (or you can see it by your self using `#lint` in the end of the file), this should be a lemma.